traffic_ops/app/db/migrations/20161208000000_alter_cdn_name_not_null.sql

```diff
@@ -0,0 +1,23 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE CDN ALTER COLUMN NAME SET NOT NULL;
+
+-- +goose Down
+-- SQL section 'Down' is executed when this migration is rolled back
+ALTER TABLE CDN ALTER COLUMN NAME SET NULL;
```

Comment:
This is because I'm creating a FK reference from snapshot.cdn to cdn.name and I was getting an error that said I could not create a FK reference to a nullable field. I can't think of any scenario where we'd create a cdn with no name....so imo this constraint should have been there from the beginning.

https://github.com/mitchell852/incubator-trafficcontrol/blob/503823b562e76bfee9a15d3ad7a426ab7b371d8b/traffic_ops/app/db/migrations/20161208000001_create_snapshots.sql#L21